### ISSUE TITLE
[DataCap Application] <TOTAL IDC> - <theskyzone.net>

### ISSUE THREAD

**Issue by fildata** [fidlabs/Enterprise-Data-Pathway#22]
### Version

1

### DataCap Applicant

TOTAL IDC

### Project ID

theskyzone

### Data Owner Name

TOTAL IDC

### Data Owner Country/Region

Korea, Republic of

### Data Owner Industry

IT & Technology Services

### Website

https://theskyzone.net/

### Social Media Handle

https://theskyzone.net/

### Social Media Type

Other

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

20GiB

### Number of replicas to store

7

### Weekly allocation of DataCap requested

100TiB

### On-chain address for first allocation

f3vwljfwtqnj4yxg6rpwq3zr535u3z6ikwxz4hcj5tddbv5ynczracmfhfi3upm2p3my7gmx7dguzmwl54ypxa

### Data Type of Application

Private Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Organization : Total IDC co., Ltd.
• Total IDC is a corporation that develops IT services in Korea and provides services to customers.
• Total IDC operates and manages its own storage data center. And based on this, we provide services to our customers.

Project : theskyzone.net
• We are providing our customers with a cloud storage service.
• Our customers are using the service for work, personal work, and backup purposes.
• Cloud storage service provides services through various channels based on web browsers, folder synchronization of Windows and Mac PCs, and iOS and Android apps.

• theskyzone.net service demo
  You can use the service demo through the information below.
  URL : https://theskyzone.net
  Demo ID : demo
  Demo Password : demo17291729*
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
We have plans to store the material stored by users on theskyzone.net in Filecoin for the long term backup. We plan to use Filecoin to periodically back-up the entire data of theskyzone.net project(User-saved data and DB backup, etc.)
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

Korea, Republic of

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
• The source of the dataset is the material stored by the user on theskyzone.net.
• Users mainly use theskyzone.net for video editing, design work, photography work, and publishing work, etc.
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
No.
```

### Please share a sample of the data

```text
https://theskyzone.net/index.php/s/x73jT6BP3PnwEgf
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [ ] I confirm

### If you chose not to confirm, what was the reason

```text
Most of the data we will store in Filecoin are the personal files of theskyzone.net users. Also, these are theskyzone.net's Database backup files, so we want them to be stored privately.
```

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Partners, Others

### If you answered "Others" in the previous question, what is the tool or platform you used

```text
filswan
cidgravity
```

### Please list the provider IDs and location of the storage providers you will be working with.

```text
provide ID, City, Country, Organization
f01115949, Seoul, Korea
f01900525, Seoul, Korea
f01770778, GyuyngGi, Korea
f01924824, GyuyngGi, Korea
f01924891, Seoul, Korea
f01927163, Seoul, Korea
f01784916, DaeGu, Korea
f01340400, Asan, Korea

Those SPs are the Korean SP Association (cooperative).
Among those miners, we plan to store our data mainly in miners that are ready to store data((initial pledge). Also, we plan to use filswan and cidgravity together from time to time.
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by fildata** [comment]
@kevzak

Hi @kevzak  please review this application. thanks.

**Comment by kevzak** [comment]
@fildata Thank you for applying.

Next steps required are:
-Github User KYC Check (https://filplus.storage/) and login and select KYC from drop down.
-KYB Form ([LINK](https://form.jotform.com/240786057753667))
-SP details. Miner IDs, entity names, locations

Once complete, you will be eligible for 50TiBs of DataCap as a first time user in this pathway. After we can review and increase toward your total amount requested.

**Comment by kevzak** [comment]
I confirm KYC check was complete - [see link](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/311#issuecomment-2051278607)

**Comment by kevzak** [comment]
Yes, I can confirm KYB was completed successfully using synaps.io , thanks @fildata

**Comment by kevzak** [comment]
One more question @fildata - the expected size of the dataset is 20GiB?

**Comment by fildata** [comment]
> How big is the size of all files when combined to equal the dataset size?
> 
> You are asking for 5 PiBs and storing 7 copies. 7 copies of what amount of dataset size? That is this field amount I need to know. Not 20 GiBs.

@kevzak 

Oh... Misunderstood.  I thought you were talking about size of a single file.
The total data size is approximately 400TB now and is increasing every day.

**Comment by kevzak** [comment]
@fildata 50TiBs was provided as your first allocation. We can review when you finished using this amount for the next allocation.

**Comment by futuremaster777** [comment]
> @fildata 50TiBs was provided as your first allocation. We can review when you finished using this amount for the next allocation.

@kevzak Thanks for your support.

**Comment by kevzak** [comment]
checker:manualTrigger

**Comment by kevzak** [comment]
SPs taking deals from report:
Provider | Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals
f01924891 | Seoul, Seoul, KRKorea Telecom | 2.72 TiB | 11.80% | 2.69 TiB | 1.15%
f01927163 | Seoul, Seoul, KRKorea Telecom | 2.69 TiB | 11.67% | 2.66 TiB | 1.16%
f01770778 | Seoul, Seoul, KRLG DACOM Corporation | 5.88 TiB | 25.51% | 5.84 TiB | 0.53%
f01924824 | Incheon, Incheon, KRLG DACOM Corporation | 5.84 TiB | 25.37% | 5.81 TiB | 0.53%
f01900525 | Seoul, Seoul, KRSK Broadband Co Ltd | 3.28 TiB | 14.25% | 3.28 TiB | 0.00%
f01115949 | Seoul, Seoul, KRSK Broadband Co Ltd | 2.63 TiB | 11.40% | 2.59 TiB | 1.19%

SPs are matching from list provided up front

provide ID, City, Country, Organization
**f01115949, Seoul, Korea**
**f01900525, Seoul, Korea**
**f01770778, GyuyngGi, Korea**
**f01924824, GyuyngGi, Korea**
**f01924891, Seoul, Korea**
**f01927163, Seoul, Korea**
f01784916, DaeGu, Korea
f01340400, Asan, Korea

**Comment by kevzak** [comment]
We will continue allocating datacap @fildata - after your initial allocation you are not eligible for 5% of your total. 250TiBs.

**Comment by fildata** [comment]
checker:manualTrigger

**Comment by fildata** [comment]
checker:manualTrigger

**Comment by fildata** [comment]
hello. @kevzak 
All datacap used. Check plz.
Thanks

**Comment by kevzak** [comment]
checker:manualTrigger

**Comment by kevzak** [comment]
Distribution is looking good so far toward 7 copies


**SPs taking deals per report**
Provider | Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals | Mean Spark Retrieval Success Rate 7d
f0137489 | Rickinghall, England, GBDaisy Corporate Services Trading Ltd | 32.00 GiB | 0.01% | 32.00 GiB | 0.00% | 87.23%
**f01924891 | Seoul, Seoul, KRKorea Telecom | 60.63 TiB | 21.26% | 60.59 TiB | 0.05% | 63.90%**
**f01927163 | Seoul, Seoul, KRKorea Telecom | 48.66 TiB | 17.06% | 48.63 TiB | 0.06% | 54.70%**
f0717969 | Burbank, California, USKrypt Technologies | 352.00 GiB | 0.12% | 352.00 GiB | 0.00% | 0.00%
f01924824 | Incheon, Incheon, KRLG DACOM Corporation | 61.53 TiB | 21.58% | 61.06 TiB | 0.76% | 40.13%
**f01770778 | Seoul, Seoul, KRLG DACOM Corporation | 43.72 TiB | 15.33% | 43.69 TiB | 0.07% | 75.55%**
f02860778 | Seoul, Seoul, KRLG DACOM Corporation | 64.00 GiB | 0.02% | 64.00 GiB | 0.00% | 0.00%
f01888808 | Bucharest, București, RORCS & RDS SA | 64.00 GiB | 0.02% | 64.00 GiB | 0.00% | 0.00%
f01900525 | Seoul, Seoul, KRSK Broadband Co Ltd | 41.78 TiB | 14.65% | 41.78 TiB | 0.00% | 40.13%
**f01115949 | Seoul, Seoul, KRSK Broadband Co Ltd | 28.34 TiB | 9.94% | 28.28 TiB | 0.22% | 69.20%**


Please explain SPs not in bold - as they were not on original list. @fildata miner ID, entity, location

**Comment by fildata** [comment]
> Distribution is looking good so far toward 7 copies
> 
> **SPs taking deals per report** Provider | Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals | Mean Spark Retrieval Success Rate 7d f0137489 | Rickinghall, England, GBDaisy Corporate Services Trading Ltd | 32.00 GiB | 0.01% | 32.00 GiB | 0.00% | 87.23% **f01924891 | Seoul, Seoul, KRKorea Telecom | 60.63 TiB | 21.26% | 60.59 TiB | 0.05% | 63.90%** **f01927163 | Seoul, Seoul, KRKorea Telecom | 48.66 TiB | 17.06% | 48.63 TiB | 0.06% | 54.70%** f0717969 | Burbank, California, USKrypt Technologies | 352.00 GiB | 0.12% | 352.00 GiB | 0.00% | 0.00% f01924824 | Incheon, Incheon, KRLG DACOM Corporation | 61.53 TiB | 21.58% | 61.06 TiB | 0.76% | 40.13% **f01770778 | Seoul, Seoul, KRLG DACOM Corporation | 43.72 TiB | 15.33% | 43.69 TiB | 0.07% | 75.55%** f02860778 | Seoul, Seoul, KRLG DACOM Corporation | 64.00 GiB | 0.02% | 64.00 GiB | 0.00% | 0.00% f01888808 | Bucharest, București, RORCS & RDS SA | 64.00 GiB | 0.02% | 64.00 GiB | 0.00% | 0.00% f01900525 | Seoul, Seoul, KRSK Broadband Co Ltd | 41.78 TiB | 14.65% | 41.78 TiB | 0.00% | 40.13% **f01115949 | Seoul, Seoul, KRSK Broadband Co Ltd | 28.34 TiB | 9.94% | 28.28 TiB | 0.22% | 69.20%**
> 
> Please explain SPs not in bold - as they were not on original list. @fildata miner ID, entity, location

Hello @kevzak 
thanks.. 

f0137489  f0717969  f02860778  f01888808 <== these SPs are found on filswan and filecoin station (twitter @filecoin station).

rest of them are Korean SP Association (cooperative). already listed when apply this application. you can see above this application and blelow attached image.
![image](https://github.com/fidlabs/Enterprise-Data-Pathway/assets/90758690/1beb0b8f-0a19-4c51-887f-99c7b909e2cf)

**Comment by kevzak** [comment]
ok thank you for clarifying

**Comment by kevzak** [comment]
15% (750TiB) granted this round

**Comment by fildata** [comment]
checker:manualTrigger

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by kevzak** [comment]
how is this project progressing @fildata? Can you provide a quick update?

**Comment by fildata** [comment]
> how is this project progressing @fildata? Can you provide a quick update?
@kevzak  

hello Kevein. Sorry for late reply. it was sleeping time here. 
The project is going well. 
We have exhausted all of our datacaps, so please add more. 

thanks.

p.s) 
I have a question. 
In the case of general Filplus, need to relicate in at least 4 SPs. Is it the same rule for E-filplus? 
I heard from someone recently that in the case of E-filplus, only need to replicate in 2 SPs. Is it right?

**Comment by kevzak** [comment]
Yes, per our [Enterprise pathway guidelines](https://github.com/fidlabs/Enterprise-Data-Pathway), we require 2 copies minimum, it can be more.

**Comment by kevzak** [comment]
I will approve the third allocation 1.5PiB (30%), however at the moment I do not have enough DataCap to give. I can give .5PiB and the remaining will come soon.

**Comment by kevzak** [comment]
checker:manualTrigger

**Comment by fildata** [comment]
> I will approve the third allocation 1.5PiB (30%), however at the moment I do not have enough DataCap to give. I can give .5PiB and the remaining will come soon.

@kevzak 

okay.. thakns a lot.

**Comment by kevzak** [comment]
checker:manualTrigger

**Comment by futuremaster777** [comment]
checker:manualTrigger

**Comment by fildata** [comment]
@kevzak 

hello Kevin. how are you? almost used DC. refill plz.

**Comment by martplo** [comment]
@fildata Hello, I wanted to let you know that I’ll be handling your allocations moving forward.

**Comment by dampud** [comment]
checker:manualTrigger

**Comment by dampud** [comment]
checker:manualTrigger
